source/_integrations/smappee.markdown

```diff
@@ -20,20 +20,47 @@ ha_platforms:
   - switch
 ---
 
-The Smappee integration will allow users to integrate their Smappee monitors, plugs and switches into Home Assistant using the [official API](https://smappee.atlassian.net/wiki/spaces/DEVAPI/overview) or the limited local API.
+The Smappee integration will allow users to integrate their Smappee monitors, Smappee Comfort Plugs and Smappee Switches into Home Assistant using the local API, local MQTT broker or the [official cloud API](https://smappee.atlassian.net/wiki/spaces/DEVAPI/overview).
+
+
+
+
+## Smappee monitors
+The table below summarizes the available options to initiatie the Smappee integration in Home Assistant per Smappee monitor type.
+
+|Monitor type|Local discovery|Cloud API|
+|---|---|---|
+|Smappee Energy|Yes|Yes|
+|Smappee Solar|Yes|Yes|
+|Smappee Plus|Yes|Yes|
+|Smappee Pro|Yes|Yes|
+|Smappee Genius|Yes|Yes|
+|Smappee Connect|No|Yes|
+|Smappee Wi-Fi Connect|No|Yes|
+|Smappee P1/S1 module|No|Yes|
+
+
+
 
 ## Local discovery
 
-In most cases, the Smappee legacy devices (Energy and Solar) will be automatically discovered by Home Assistant through network scanning.
+In most cases, the Smappee Energy, Solar, Plus, Pro and Genius monitors will be automatically discovered by Home Assistant through network scanning.
 Those automatically discovered Smappee devices are listed on the integrations page and can be configured without any additional details.
-This will provide you a limited number of entities only (realtime power values, solar if applicable and the installed Smappee Comfort Plugs and Switches).
+This will provide you a limited number of entities only.
 If your home network doesn't support mDNS you can still manually initiate the Smappee integration by choosing the LOCAL option and entering the IP address of the Smappee monitor through the configuration flow.
 
-If you have API credentials, we strongly suggest you use the Smappee cloud configured integration by following the scheme below.
+#### Sensor
+A sensor entity is being added for the current active power usage. In case of solar production an entity for the active power production is added as well.
```

Comment:
```suggestion
A sensor entity is being added for the current active power usage. In case of solar production, an entity for active power production is added as well.
```